content/src/ports/activeEntities.ts

```diff
@@ -18,6 +23,10 @@ export type ActiveEntities = {
    * Note: result is cached, even if the pointer has no active entity
    */
   withPointers(pointers: Pointer[]): Promise<Entity[]>
+  /**
+   * Retrieve active entities which their pointers match the given urn prefix
+   */
+  withPrefix(urnPrefix: string): Promise<{ pointer: Pointer; entityId: EntityId }[]>
```

Comment:
What does `urnPrefix` accepts? because `something:%` is very SQL-ish. We may want to accept only tailing wildcards like `something:*` for URNs and perform a sanitization thinking in future non-sql catalysts API compliance

Reply:
Also, % is hard to write in a query param `%25`

The URL would look like this
- https://peer.decentraland.org/entities/currently-pointed/urn:decentraland:off-chain:base-wearables:%25
- vs https://peer.decentraland.org/entities/currently-pointed/urn:decentraland:off-chain:base-wearables:*

Reply:
Since the wildcard is being appended in the SQL query

Reply:
Right, I missed the appended %. 

I think we must sanitize the prefix regardless

Reply:
Will do in the controller with a regex I think

Reply:
According to the documentation of `sql-template-strings`: https://github.com/felixfbecker/node-sql-template-strings#sql-template-strings

Doing https://github.com/decentraland/catalyst/pull/969/files#diff-72c9f3491f1be652887175eacf28a18e8d4e626c378986ceded36ccc179d7a20R11 is enough